ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -424,15 +424,16 @@ tests:
     - chain: cucushift-installer-check-cluster-health
   timeout: 4h0m0s
 - always_run: false
-  as: e2e-aws-ovn-user-provisioned-dns
+  as: e2e-aws-custom-dns-techpreview
   optional: true
   steps:
-    cluster_profile: aws
+    cluster_profile: aws-qe
     env:
       AWS_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
+      BASE_DOMAIN: qe.gcp.devcluster.openshift.com
       FEATURE_SET: TechPreviewNoUpgrade
```

Comment:
This job is not updated to use the dedicated featureGate to run the installation like what gpei posted in the last comment, right ?